blog/2024-08-14-conda-ecosystem-explained.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: "Conda Ecosystem Explained"
+slug: "2024-08-14-conda-ecosystem-explained"
+authors: [dashagurova, jezdez]
+tags: [conda, community, governance]
+description: |
+  Get to know the conda ecosystem—a vibrant community supporting an open-source, language-agnostic, and multi-platform package management system. Discover conda CLI tools for managing packages and environments, and explore key repositories like conda-forge, bioconda and defaults. You will learn how to get started with conda tools, the important things to keep in mind, and how multistakeholder governance ensures the ecosystem remains reliable and accessible to all.
+---
+
```

Comment:
I would add the same text as in `description` above here as the introduction of the full blog post, and then `<!-- truncate -->`. Otherwise you get the full text in the [blog list](https://deploy-preview-198--conda-dot-org.netlify.app/blog) and is a bit too long for that.

```suggestion

Get to know the conda ecosystem—a vibrant community supporting an open-source, language-agnostic, and multi-platform package management system. Discover conda CLI tools for managing packages and environments, and explore key repositories like conda-forge, bioconda and defaults. You will learn how to get started with conda tools, the important things to keep in mind, and how multistakeholder governance ensures the ecosystem remains reliable and accessible to all.

<!-- truncate -->
```

Reply:
Good call!